### DISCUSSION TITLE
Can't get rmview working with 2.11

### DISCUSSION THREAD

**Discussion by dustinkirkland** [bordaigorl/rmview#124] [Q&A] [answered]
Howdy!  I've not yet managed to get rM working with 2.11 here...  Can anyone confirm it's working with reMarkable's v2.11?

The desktop app is stuck on the "Connecting to reMarkable..." screen.  Here's my verbose command line debug:

```
$ rmview -v

2022-01-08 13:12:28-0600 [-] Log opened.

[INFO] STARTING: Sat Jan  8 13:12:28 2022

[INFO] Searching configuration in rmview.json, /home/kirkland/.config/rmview.json

[DEBUG] Configuration failure in rmview.json: [Errno 2] No such file or directory: 'rmview.json'

[INFO] Fetching configuration from /home/kirkland/.config/rmview.json

[DEBUG] Config values: {'ssh': {'address': '10.0.0.89', 'password': '**********'}, 'backend': 'screenshare', 'orientation': 'auto', 'pen_size': 15, 'pen_color': 'red', 'pen_trail': 200}

[INFO] Using known hosts file: /home/kirkland/.config/rmview_known_hosts

[INFO] Loaded known hosts from /home/kirkland/.config/rmview_known_hosts

[INFO] Connecting...

[INFO] Connected to 10.0.0.89

[INFO] Detected reMarkable 2.0

[INFO] Using backend 'screenshare'

[INFO] Connecting to ScreenShare, make sure you enable it

[WARNING] Authenticating, please wait...

2022-01-08 13:12:29-0600 [-] ChallengeReaderProtocol starting on 5901

2022-01-08 13:12:29-0600 [-] Starting protocol <rmview.screenstream.screenshare.ChallengeReaderProtocol object at 0x7f8a710a65e0>

[DEBUG] Stopping connection worker
```

**Comment by bordaigorl** [comment, marked as answer]
It works on 2.11 for me.
Have you started the screen sharing from the tablet?
If so you may also want to check #111 and make sure you opened port 5901.

**Comment by dustinkirkland** [comment]
Thanks, that's exactly it.  I was able to make rmview work just fine, on a different machine, with a simpler network setup.

I still don't have it working on my primary machine (which is ethernet connected on the same network, but I'm utilizing a br0 on that system as it's a host for other containers/VMs).  That machine just simply isn't receiving the challenge on UDP 5901, so I'm still debugging that...
